File: src/dlls/CSSPModels/srcweb/WebMWQMRun.cs
/*
 * Manually edited
 * 
 */
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace CSSPModels
{
    [NotMapped]
    public partial class WebMWQMRun
    {
        #region Properties
        public List<WebBase> TVItemParentList { get; set; }
        public List<MWQMRunModel> MWQMRunModelList { get; set; }
        #endregion Properties

        #region Constructors
        public WebMWQMRun()
        {
            TVItemParentList = new List<WebBase>();
            MWQMRunModelList = new List<MWQMRunModel>();
        }
        #endregion Constructors
    }

    [NotMapped]
    public partial class MWQMRunModel : WebBase
    {
        #region Properties
        public MWQMRun MWQMRun { get; set; }
        public List<MWQMRunLanguage> MWQMRunLanguageList { get; set; }
        #endregion Properties

        #region Constructors
        public MWQMRunModel()
        {
            MWQMRun = new MWQMRun();
            MWQMRunLanguageList = new List<MWQMRunLanguage>();
        }
        #endregion Constructors
    }
}

File: src/webs/CSSPWebToolsAng/client-apps/src/app/models/webmwqmrun.ts
import { MWQMRun } from './generated/MWQMRun.model';
import { MWQMRunLanguage } from './generated/MWQMRunLanguage.model';
import { WebBase } from './webbase';

export class WebMWQMRun {
    TVItemParentList: WebBase[] = [];
    MWQMRunModelList: MWQMRunModel[] = [];
}

export class MWQMRunModel extends WebBase {
    MWQMRun: MWQMRun = new MWQMRun();
    MWQMRunLanguageList: MWQMRunLanguage[] = [];
}
